plugins/inputs/redis/redis_test.go

```diff
@@ -116,6 +116,22 @@ func TestRedis_ParseMetrics(t *testing.T) {
 	}
 	acc.AssertContainsTaggedFields(t, "redis", fields, tags)
 	acc.AssertContainsTaggedFields(t, "redis_keyspace", keyspaceFields, keyspaceTags)
+
+	cmdstatSetTags := map[string]string{"host": "redis.net", "replication_role": "master", "command": "set"}
+	cmdstatSetFields := map[string]interface{}{
+		"calls":         int64(261265),
+		"usec":          int64(1634157),
+		"usec_per_call": float64(6.25),
+	}
+	acc.AssertContainsTaggedFields(t, "redis_cmdstat", cmdstatSetFields, cmdstatSetTags)
+
+	cmdstatCommandTags := map[string]string{"host": "redis.net", "replication_role": "master", "command": "command"}
+	cmdstatCommandFields := map[string]interface{}{
+		"calls":         int64(1),
+		"usec":          int64(990),
+		"usec_per_call": float64(990.0),
+	}
+	acc.AssertContainsTaggedFields(t, "redis_cmdstat", cmdstatCommandFields, cmdstatCommandTags)
```

Comment:
Should we needs to add test in case that usec and usec_per_call are not number?

Reply:
If this is something that may be returned, let's test for and handle it.

Reply:
Is there a pattern from testutil I could use?

FWIW, I don't expect the type to change given the age of this commit :)
https://github.com/antirez/redis/commit/d7ed7fd2fcd29fbba605fdd06b9498ffd85c092f